subjects/Elf/AVR32/ls.reko/ls_seg00001000_0000.c

```diff
@@ -17161,7 +17161,7 @@ byte fn0000EB48(Eq_n r0, Eq_n r1, ui32 r2, struct Eq_n * r3, Eq_n r4, Eq_n r5, s
 					if (r2_n <= r5_n)
 					{
 						r2_n.u0 = r5_n.u0 + 1;
-						if (r5_n.u0 + 1 < r5_n)
+						if (false)
```

Comment:
This looks like a saturated add: first check for unsigned overflow, and if so, force the value to 0xFFFFFFFF. This regression cannot be allowed.